packages/corvid-e2e-tests/test/drivers/localEditorDriver.js

```diff
@@ -36,11 +35,29 @@ module.exports = page => {
   };
 
   return {
-    waitForEditor,
-    waitForLoginForm,
-    push,
-    addTextElement,
-    login,
-    close
+    close,
+
+    waitForLogin: async () => {
+      try {
```

Comment:
no need for the try/catch or for the return Promise.resolve
this will give you the same:

waitForLogin: async () => {
	await waitForLoginForm();
	return {login}
}

same below for "waitForEditor"

Reply:
actually i am using try/catch for rejecting the promise..
in the test i am expecting the promise to be rejected
`await expect(editorCloneDriver.waitForLogin()).rejects.toBeDefined();`

Reply:
an async function should return a rejected promise when an error is thrown in it.
unless i'm missing something, it should work exactly the same way

Reply:
you are correct :)